src/LinuxTracing/UprobesUnwindingVisitor.h

```diff
@@ -146,6 +152,15 @@ class UprobesUnwindingVisitor : public PerfEventVisitor {
 
   absl::flat_hash_map<pid_t, absl::flat_hash_map<uint64_t, StackSlice>>
       thread_id_stream_id_to_stack_slices_{};
+
+  // tid_to_root_namespace_tid_ holds a mapping from all tids in the target process namespace to the
+  // corresponding tids in the root namespace.
+  // We obtain the initial state of this mapping at the beginning of the capture via
+  // SetInitialTidMapping. During the capture we observe task_newtask and clone{3} tracepoints to
+  // keep track of new threads (new_task_parent_to_root_namespace_tids_ is required for this
+  // bookkeeping).
+  absl::flat_hash_map<pid_t, pid_t> tid_to_root_namespace_tid_;
+  absl::flat_hash_map<pid_t, pid_t> new_task_parent_to_root_namespace_tids_;
```

Comment:
I found this name already confusing during the previous review, and now that I understand better, I propose:
`new_task_root_namespace_parent_tid_to_root_namespace_tid_`

Reply:
done